src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -276,7 +276,7 @@ protected virtual async Task<Stream> CloneRequestBodyAsync(HttpRequest request,
         }
         else
         {
-            Logger.LogWarning("Aggregation does not support body copy without Content-Length header!");
+            Logger.LogInformation("Aggregation does not support body copy without Content-Length header, skipping body copy.");

```

Comment:
The message lacks route details. Including the route name would facilitate understanding of the failed route when reviewing the log.
Examples can be found in the repository here: [RouteName](https://github.com/search?q=repo%3AThreeMammals%2FOcelot%20RouteName&type=code).

Reply:
I added it please tell me if it's OK for you.

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/2146/commits/e76c3b44de70692468877b4409abf7b9d8253dbf ✔️